<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BARbook</class>
 <widget class="QMainWindow" name="BARbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QUẢN LÍ MƯỢN TRẢ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);
background-color: rgb(255, 228, 196);</string>
    </property>
    <property name="title">
     <string>THÔNG TIN ĐỘC GIẢ</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HỌ TÊN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GIỚI TÍNH:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtinputReaderID">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NHẬP MÃ ĐỘC GIẢ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFindReader">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 255);
background-color: rgb(255, 85, 255);</string>
     </property>
     <property name="text">
      <string>TÌM KIẾM</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbName">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbSex">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbStatusReader">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TRẠNG THÁI THẺ:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>531</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 228, 196);</string>
    </property>
    <property name="title">
     <string>SÁCH ĐỘC GIẢ ĐANG MƯỢN</string>
    </property>
    <widget class="QTableView" name="tableBookBorrow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>531</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 228, 196);</string>
    </property>
    <property name="title">
     <string>TÌM SÁCH</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NHẬP TÊN SÁCH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NHẬP TÁC GIẢ</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtinputISBN">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NHẬP MÃ ISBN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFindHeadBook">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 255);</string>
     </property>
     <property name="text">
      <string>TÌM KIẾM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NHẬP THỂ LOẠI</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtinputBookName">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtinputAuthor">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtinputTypeBook">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 228, 196);</string>
    </property>
    <property name="title">
     <string>DANH SÁCH TÌM KIẾM</string>
    </property>
    <widget class="QTableView" name="tableListBook">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>561</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>610</y>
      <width>531</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 228, 196);</string>
    </property>
    <property name="title">
     <string>THÔNG TIN SÁCH</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MÃ SÁCH:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TÊN SÁCH:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VỊ TRÍ:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TRẠNG THÁI:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBorrowBook">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 255);
background-color: rgb(255, 85, 255);</string>
     </property>
     <property name="text">
      <string>MƯỢN SÁCH</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReturnBook">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 255);</string>
     </property>
     <property name="text">
      <string>TRẢ SÁCH</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbIDBook">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbBookName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbBookLocation">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbBookStatus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1241</width>
      <height>861</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(205, 133, 63);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>790</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LỊCH SỬ MƯỢN TRẢ SÁCH GẦN NHẤT</string>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
